.github/workflows/smoke.yml

```diff
@@ -162,7 +162,15 @@ jobs:
             exit 1
         fi
 
-        ./.github/verify-apache2.sh
+        attempts=0
+        while ! ./.github/verify-apache2.sh; do
+          sleep 10
+          attempts=$((attempts+1))
+          if [ "$attempts" -eq 3 ]; then
+            echo "Apache health check timed out"
+            exit 1
+          fi
+        done
```

Comment:
Can we build the retry loop into the `verify-apache2.sh` script to avoid the cut and paste in a few places

Reply:
That could be nice if we add the number of attempts as an input parameter. Different tries indifferent sections may differ in the number of attempts.

Reply:
Good spot!